### PR TITLE
feat(facility-language-server): add facility language server

### PR DESCRIPTION
**Pull request by bryankenote**
https://facilityapi.github.io/

### CODE REVIEW

**Comment by glepnir** [comment]
fix lint please.

**Comment by bryankenote** [comment]
> fix lint please.

I'm not sure the lint errors have anything to do with my changes. I didn't change any of the files in the output. The same lint errors are happening in other prs.

**Comment by glepnir** [comment]
oh it's unrelated.

File: lua/lspconfig/server_configurations/facility_language_server.lua [hunk 1]
```diff
@@ -0,0 +1,18 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'facility-language-server' },
+    filetypes = { 'fsd' },
+    single_file_support = true,
+    root_dir = util.find_git_ancestor,
+    settings = {},
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/facility_language_server.lua:9]: can remove.

**Pull request merged**